frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -240,6 +246,21 @@ function ReaderPaging:addToMainMenu(menu_items)
             self.inverse_reading_order = not self.inverse_reading_order
             self:setupTapTouchZones()
         end,
+        hold_callback = function(touchmenu_instance)
+            self.inverse_reading_order = not self.inverse_reading_order
+            
+            self:setupTapTouchZones()
```

Comment:
Is toggling it and calling self:setupTapTouchZones() (here and in readerrolling) before the ok_callback right?
Either not at all (if settings as default should not apply to the current document), or in the ok_callback if it should?

Reply:
Good point. I think not at all would be consistent with how it works elsewhere, and also with the logic behind text string.